core/jazz_logs/swagger/swagger.json

```diff
@@ -429,22 +427,30 @@
           "type": "string",
           "description": "Environment in which the service is deployed"
         },
-		"category": {
-		  "enum": [
+        "category": {
+          "enum": [
             "api",
             "function"
           ],
           "type": "string",
           "description": "Service type"
         },
-		"start_time": {
+        "asset_type": {
+          "enum": [
+            "apilogs",
```

Comment:
Supported asset_types: https://github.com/tmobile/jazz/blob/sls_app/core/jazz_assets/config/global-config.json#L25

Reply:
Updated asset_type list in swagger file

Reply:
changes look good.